<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetRibbon</class>
 <widget class="QWidget" name="WidgetRibbon">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>237</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>编辑飘带</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <property name="sizeConstraint">
    <enum>QLayout::SetMinimumSize</enum>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>飘带名称</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="name"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>绑定骨骼</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="boneId"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>使用贴图</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QComboBox" name="textureId"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>飘带分段</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="edgePerSec"/>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>飘带长度</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="edgeLifeSec"/>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>飘带颜色</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QColorPicker" name="color"/>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>分段上限</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLineEdit" name="maxEdgeCount"/>
   </item>
   <item row="6" column="1">
    <widget class="QCheckBox" name="blendMode">
     <property name="text">
      <string>加法混合</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QColorPicker</class>
   <extends>QLineEdit</extends>
   <header location="global">QColorPicker.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
